sputnikdao2/src/proposals.rs

```diff
@@ -341,9 +341,47 @@ impl Contract {
                 receiver_id,
             } => self.internal_execute_bounty_payout(*bounty_id, &receiver_id.clone().into(), true),
             ProposalKind::Vote => PromiseOrValue::Value(()),
+        };
+        match result {
+            PromiseOrValue::Promise(promise) => promise
+                .then(ext_self::on_proposal_callback(
+                    proposal_id,
+                    env::current_account_id(),
+                    0,
+                    GAS_FOR_FT_TRANSFER,
```

Comment:
We could consider `GAS_FOR_PROPOSAL_CALLBACK` or something. Not urgent.
It does seem like this match block is how we simplify the work that was attempted in #32 👍🏼